src/components/popup/drawer/drawer-section.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import { Box, BoxProps } from 'rebass';
+
+// Components
+import Labeling from '../../typography/labeling';
+
+export interface DrawerSectionProps extends Omit<BoxProps, 'css' | 'title'> {
+  title: React.ReactNode;
```

Comment:
Force a string

Reply:
Sorry, I can't use it just as a string, because I put CLose button in the Drawer component.

Reply:
But drawer section does not has to be close, no?